### ISSUE TITLE
Suggestions - Align with ESDC Self-Paced Web Accessibility Course

### ISSUE THREAD

**Issue by chanmichaely** [cenw-wscoe/WCAG-in-Plain-Language#66]
Align with ESDC Self-Paced Web Accessibility Course

The ESDC self-paced course contains 80+ vetted pages in English and French, covering all 50 WCAG 2.1 AA criteria and some AAA. It includes examples, screenshots, sample code and references to WCAG, WET and the Canada Content Style Guide.  
 
The course should answer most users’ questions, frequent and infrequent, about WCAG.  
  
If we align the FAQ with the course, we can ensure consistency, avoid contradictions and streamline the effort. 
 

- Align categories
-- See attached categories from ESDC course 

- Use the course as a knowledge base.  
-- Link to course content in the answers. 
-- Skim the course for answers to questions: 
--- If the answer is in the course, re-use the explanation for the answer 
--- If the answer is not in the course, submit GitHub issues for any missing or incorrect info in the course 

- We are currently looking into adding search to the course 

- Consider hosting the FAQ in the course appendix:  

See attached categories from ESDC course:
[Categories from the ESDC Self.docx](https://github.com/cenw-wscoe/WCAG-in-Plain-Language/files/9260628/Categories.from.the.ESDC.Self.docx)
